### DISCUSSION TITLE
Lazy & Masson windows full screen

### DISCUSSION THREAD

**Discussion by edshamis** [LazyVim/LazyVim#429] [Q&A] [answered]
When on Termux I'd like to have these kind of UI full screen
![Screenshot_2023-03-14-07-41-37-442_com.termux.jpg](https://user-images.githubusercontent.com/47914518/224909070-27c84581-9670-4fa4-8f74-ec3c3322c966.jpg)
![Screenshot_2023-03-14-07-39-37-633_com.termux.jpg](https://user-images.githubusercontent.com/47914518/224909097-bd9cf170-a193-4f55-ba41-1e4f4485d2bc.jpg)

Can you please help me figure out the right settings?

**Comment by folke** [comment, marked as answer]
Have you tried reading the docs for lazy.nvim and for mason.nvim? It's in there

**Reply by edshamis** [reply]
Was looking for it in Noice for some reason. Thanks for pointing out.
Appreciate your work and your excellent docs 🙏
